### PR TITLE
Updated handbook with recommended software

### PR DESCRIPTION
**Pull request by javierM84**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -332,12 +332,21 @@ const fooCompany = Immutable.Map({
...
 - [Caching your GitHub password](https://help.github.com/articles/caching-your-github-password-in-git/)
 
+## Recommended Software
+
+- IDE: Atom / Nuclide
+- Git UI: SourceTree
+- API requests: Postman
+- GitHub Notifications : Gitify
+- MongoDB manager: Robomongo
+- SQLite manager: MesaSQLite
```
Comment (DaleJefferson) [inline README.md]: What about https://www.sequelpro.com/

File: README.md [hunk 2]
```diff
@@ -332,12 +332,21 @@ const fooCompany = Immutable.Map({
 const updatedFooCompany = fooCompany.set('isRegistered', true)
 
 ```
-
 ## Tips
 
 - [View all PRs by RNF](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3ARNFDigital+sort%3Aupdated-desc)
 - [Caching your GitHub password](https://help.github.com/articles/caching-your-github-password-in-git/)
 
+## Recommended Software
```
Comment (jdmunro) [inline README.md]: Don't forget to add to the table of contents - it'd also be nice to link to the software you mention.

File: README.md (hunk 1 above)
Comment (javierM84) [inline README.md]: SequelPro is more for relational databases ( mysql) I will include it

File: README.md (hunk 2 above)
Comment (javierM84) [inline README.md]: Uuups, yes I forgot that

**Pull request merged**